CIP-xxxxx/README.md

```diff
@@ -0,0 +1,216 @@
+---
+CIP: ???
+Title: Web-Wallet Bridge - Plutus wallets
+Status: Proposed
+Category: Wallets
+Authors: 
+  - Leo
+Implementors: NA
+Discussions:
+Created: 2023-10-12
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This document describes a CIP-30 extension allowing webpages (i.e., DApps) to interface with Cardano Plutus wallets. This interface is a work in progress, and new versions are expected to be included as the ecosystem evolves. 
+
+## Motivation: why is this CIP necessary?
+
+In order to facilitate future DApp development, we will need a way for DApps to communicate with Plutus wallets, given the unique complexities of Plutus-based addresses. Special provisions need to be made to make the connector compatible with them.  
+
+Specifically, apps building transactions need to be able to get the following information from the wallet:
+
+- Script descriptor
+- `ScriptRequirements`
+- Change Datum
+
+Additionally, apps need to be able to submit a transaction to the wallet for signing in an asynchronous manner, as gathering of signatures can take a long time and each wallet provider will have its own way of handling this process. 
+
+Finally, the `signData()` endpoint will have to be disabled when using this extension since they are not compatible with Plutus-based addresses. Plutus contracts cannot sign data, only validate transactions.
+
+### Rationale for the required data
+
+- Script descriptor:
+    Any transaction consuming a UTXO from a Plutus-based address must attach the corresponding script. 
+
+- `ScriptRequirements`:
+    The `TxContext` that is required to be able to validate the transaction. It encompasses all the possible combinations of requirements for the transaction to be valid, as such it is represented by an array of `ScriptRequirement` objects.
+
+- Change Datum:
+    The datum that will be used as the change output for the transaction. This is required for wallets based on Plutus V2 and before, as the change output must contain a datum to be valid and spendable.
```

Comment:
```suggestion
- Script descriptor
  - Any transaction consuming a UTXO from a Plutus-based address must attach the corresponding script. 
- `ScriptRequirements`
  - The `TxContext` that is required to be able to validate the transaction. It encompasses all the possible combinations of requirements for the transaction to be valid, as such it is represented by an array of `ScriptRequirement` objects.
- Change Datum
  - The datum that will be used as the change output for the transaction. This is required for wallets based on Plutus V2 and before, as the change output must contain a datum to be valid and spendable.

Additionally, apps need to be able to submit a transaction to the wallet for signing in an asynchronous manner, as gathering of signatures can take a long time and each wallet provider will have its own way of handling this process. 

Finally, the `signData()` endpoint will have to be disabled when using this extension since they are not compatible with Plutus-based addresses. Plutus contracts cannot sign data, only validate transactions.
```
Those present at yesterday's meeting got a bit dizzy thinking there were 2 _Rationale_ document sections & flipping back & forth.  These 3 items below aren't "rationale" as we've defined it in the CIP process (comparative reasons why design decisions were taken).

I think this problem could be fixed & would produce a more readable CIP if each of these 3 items were moved up with the corresponding item above.  I'm taking a shot at correcting this & I'll commit it to confirm it's readable & feel free to make further changes afterward.